x-pack/plugins/index_management/server/plugin.ts

```diff
@@ -31,21 +31,26 @@ export class IndexMgmtServerPlugin implements Plugin<IndexManagementPluginSetup,
     this.config = initContext.config.get();
   }
 
-  setup(
-    { http, getStartServices }: CoreSetup,
-    { features, security }: Dependencies
-  ): IndexManagementPluginSetup {
+  setup({ http }: CoreSetup, { features, security }: Dependencies): IndexManagementPluginSetup {
     features.registerElasticsearchFeature({
       id: PLUGIN.id,
       management: {
         data: ['index_management'],
       },
       privileges: [
+        {
+          requiredClusterPrivileges: ['manage_enrich'],
+          ui: ['manageEnrich'],
+        },
+        {
+          requiredClusterPrivileges: ['manage_index_templates'],
+          ui: ['manageIndexTemplate'],
+        },
         {
           // manage_index_templates is also required, but we will disable specific parts of the
           // UI if this privilege is missing.
           requiredClusterPrivileges: ['monitor'],
-          ui: [],
+          ui: ['enabled'],
```

Comment:
Just for my understanding, why is this change needed?

Reply:
Either there's a single value and `ui: [],` is okay or it needs a value so there can be other values as well.